src/main/java/com/xebia/headerbuddy/annotations/validators/ValidEmailValidator.java

```diff
@@ -0,0 +1,22 @@
+package com.xebia.headerbuddy.annotations.validators;
+
+import com.xebia.headerbuddy.annotations.ValidEmail;
+
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+
+/*
+ * Does nothing at the moment but if you want greater url validation e.g. give some sites not allowance you can create a validate in the  isValid method
```

Comment:
If it does nothing, why would it be here? I understand that you'd want to keep it for future implementations but what if that never comes around? I suggest deleting it if it's not used at this moment or near future.